nupic/embodied/agents/curious_ppo_agent.py

```diff
@@ -401,12 +401,14 @@ def learn(self):
         # Update the networks & get losses for nepochs * nminibatches
         print("Update the models")
         for idx in range(self.nepochs):
-            print(f"Starting epoch: {idx+1}/{self.nepochs}")
+            if self.debugging:
```

Comment:
This looks good. Once we change to use logging, this can be loggin.debug - then the command line argument can set the logging level globally, so no ifs are required.